packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-card.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Text } from '@deriv/components';
 import { CFD_PLATFORMS } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
-import { TModifiedTradingPlatformAvailableAccount } from 'Components/props.types';
+import { TModifiedTradingPlatformAvailableAccount, TCompareAccountsCard } from 'Components/props.types';
```

Comment:
TModifiedTradingPlatformAvailableAccount is not usable anymore in the file, so kindly remove it